packages/react-codemirror/src/CypherEditor.tsx

```diff
@@ -69,6 +69,12 @@ export interface CypherEditorProps {
    * @default true
    */
   lint?: boolean;
+  /**
+   * Whether the editor should perform signature help.
+   *
+   * @default true
+   */
+  signatureHelp?: boolean;
```

Comment:
Do you mean the error linting? the normal syntax coloring can't be turned off. The reason for the linter setting is to disable it by default on big version mismatches (for example 3.5) and to a lesser extent because some people don't like linting

For the case of not having the editor connected to a particular database, then you'd not provide function signatures, meaning the feature would effectively be turned off regardless. I don't see the case where you bother to pass in the signatures, but don't want them

Reply:
Decided to remove the flag